aixtend/factories/model_factory.py

```diff
@@ -74,40 +72,6 @@ def create_asset_from_id(cls, model_id: str) -> Model:
             logging.error(message)
             raise Exception(f"Status {status_code}: {message}")
 
-    @classmethod
-    def subscribe_to_asset(cls, model: Model) -> None:
```

Comment:
Clarification: Does this mean one doesn't need to subscribe to an asset?

Reply:
User no longer needs to subscribe. Concept of subscription has been removed altogether